src/github.com/mihard/behat-parallel-runner/main.go

```diff
@@ -53,9 +53,9 @@ func main() {
 		res := <-rc
 
 		if res.Ok {
-			log.Printf("%s %s... OK\n", res.File, res.Scenario.Scenario)
+			log.Printf("%s [Done in %s] %s... OK \n", res.Scenario.File, res.ExecutionTime, res.Scenario.Scenario)
```

Comment:
Please use `res.ExecutionTime.Seconds()`, add the time measure to the template and use `%.2f` instead `%s` for the time.

Reply:
And the same for the line 58